itests/src/test/java/fi/metatavu/edelphi/test/ui/base/panel/PanelAdminDashboardTestsBase.java

```diff
@@ -0,0 +1,172 @@
+package fi.metatavu.edelphi.test.ui.base.panel;
+
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+
+import fi.metatavu.edelphi.test.mock.PanelMocker;
+import fi.metatavu.edelphi.test.ui.base.AbstractUITest;
+
+public class PanelAdminDashboardTestsBase extends AbstractUITest {
+  
+  
+  private static final String QUERY_ACTIVITY_SELECTOR = "#panelAdminDashboardProcessLeftBlock .feature:nth-of-type(1)";
+  private static final String QUERY_RESULTS_SELECTOR = "#panelAdminDashboardProcessLeftBlock .feature:nth-of-type(2)";
+  private static final String COMPARE_REPORTS_SELECTOR = "#panelAdminDashboardProcessLeftBlock .feature:nth-of-type(3)";
+
+  private static final String PANEL_STAMPS_SELECTOR = "#panelAdminDashboardProcessRightBlock .feature:nth-of-type(1)";
+  private static final String PANEL_EXPERTS_SELECTOR = "#panelAdminDashboardProcessRightBlock .feature:nth-of-type(2)";
+  private static final String SEND_EMAIL_SELECTOR = "#panelAdminDashboardProcessRightBlock .feature:nth-of-type(3)";
+  private static final String PANEL_BULLETINS_SELECTOR = "#panelAdminDashboardProcessRightBlock .feature:nth-of-type(4)";
+  
+  private static final String INVITE_FEATURE_SELECTOR = "#panelAdminDashboardUsersBlockContent .feature:nth-of-type(1)";
+  private static final String PANEL_USERS = "#panelAdminDashboardUsersBlockContent .feature:nth-of-type(2)";
+  private static final String PANEL_USER_GROUPS = "#panelAdminDashboardUsersBlockContent .feature:nth-of-type(3)";
+
+  private PanelMocker panelMocker;
+  
+  @Before
+  public void before() {
+    panelMocker = new PanelMocker();
+  }
+  
+  @After
+  public void after() {
+    panelMocker.cleanup();
+  }
+
+  @Test
+  public void testFeaturesBasic() {
+    login(ADMIN_EMAIL);
+    createTestPanel();
+    
+    updateUserSubscription(1l, "BASIC", toDate(2016, 1, 1), toDate(2250, 10, 10));
+    waitAndClick(".panel .GUI_navigation .menu .menuItem:nth-child(2) a");
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png 'Severity: CRITICAL') Define a constant instead of duplicating this literal ".panel .GUI_navigation .menu .menuItem:nth-child(2) a" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1192)